### ISSUE TITLE
Firefox 57 removes Arc Theme

### ISSUE THREAD

**Issue by Zeitwaechter** [horst3180/arc-firefox-theme#136]
Like expected this stopped working with Firefox 57.

At least the normal FF57 looks still awesome thanks to Arc-Darker Gnome/Cinnamon Theme <3

**Comment by ArchangeGabriel** [comment]
Yeah, the default theme uses the system GTK one, so it fits well. :)

**Comment by eigan** [comment]
But the GTK theme adds color on all form inputs too

**Comment by ArchangeGabriel** [comment]
eigan: That is https://bugzilla.mozilla.org/show_bug.cgi?id=70315.
